compat/mingw.h

```diff
@@ -501,6 +501,9 @@ static inline void convert_slashes(char *path)
 #define find_last_dir_sep mingw_find_last_dir_sep
 int mingw_offset_1st_component(const char *path);
 #define offset_1st_component mingw_offset_1st_component
+struct strbuf;
+int mingw_is_mount_point(struct strbuf *path);
+#define is_mount_point mingw_is_mount_point
```

Comment:
Does the declaration of `mingw_is_mount_point()` not belong into b494e36550441f241c69c5c6c1c06fd846be8655? I think the latter commit should be amended by this hunk.